services/QuillLMS/app/controllers/api/v1/activities_controller.rb

```diff
@@ -44,12 +53,11 @@ def create
   end
 
   def destroy
-    if @activity.destroy!
```

Comment:
Haha.  I hadn't noticed that this `destroy!` call had an `if/else` wrapped around it that, obviously, could never run the `else` branch because failure to `destroy!` raises an exception.  Nice catch.